app/blueprints/assessments/helpers.py

```diff
@@ -98,6 +98,29 @@ def set_application_status_in_overview(application_overviews):
     return application_overviews
 
 
+def set_assigned_info_in_overview(application_overviews, users_for_fund):
+    users_for_fund_dict = {user["account_id"]: user for user in users_for_fund}
```

Comment:
```suggestion
users_for_fund_dict = {user["account_id"]: user for user in users_for_fund} if users_for_fund else {}
```

Reply:
ran into an error page locally when there are no users in the account store for the round (might not be a common scenario but this avoids this error page)

Reply:
Oh yep good shout, will update.